Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/examples.txt

```diff
@@ -15,4 +15,9 @@ cs-falcon-update-ioa-exclusion id=123456 description=demo-description
 cs-falcon-delete-ioa-exclusion id=123456
 cs-falcon-search-ioa-exclusion limit=1
 cs-falcon-list-quarantined-file limit=1
-cs-falcon-apply-quarantine-file-action filename=test.exe action=delete comment=demo-comment
\ No newline at end of file
+cs-falcon-apply-quarantine-file-action filename=test.exe action=delete comment=demo-comment
+cs-falcon-ods-query-scan
```

Comment:
examples should be added with the proper arguments. Please regenerate readme after fix

Reply:
np